ext/hal/altera/CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ if(CONFIG_ALTERA_AVALON_QSPI)
   add_subdirectory(drivers/altera_epcq_controller/HAL/src)
 endif()
 
-if(CONFIG_ALTERA_AVALON_QSPI)
```

Comment:
what happened to the QSPI driver?

Reply:
actually, this was a typo which was done previously. Now i simply fixed it. The QSPI usage is just above this block.